### PR TITLE
Fix/sniff wiki basic action other actions

### PR DESCRIPTION
**Pull request by ilyeskallala**

### CODE REVIEW

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/WikiBasicActionOtherActionsTestIT.java [hunk 1]
```diff
@@ -500,4 +511,619 @@ public void test13_ExportPageAsPDF() {
     wikiHomePage.deleteWiki(title);
   }
 
+  /**
+   * <li>Case ID:122811.</li>
+   * <li>Test Case Name: Watch a wiki page of intranet.</li>
+   * <li>Pre-Condition:</li>
+   * <li>Post-Condition:</li> check issue WIKI-976 (wrong link in the mail)
+   */
+  /*
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/WikiBasicActionOtherActionsTestIT.java]: No need to have 2 blocks of comments (this remark is valid for all comments blocks of this PR).

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/WikiBasicActionOtherActionsTestIT.java [hunk 2]
```diff
@@ -500,4 +511,619 @@ public void test13_ExportPageAsPDF() {
...
+    homePagePlatform.goToWiki();
+    wikiHomePage.goToAPage(title);
+    wikiHomePage.goToEditPage();
+    richTextEditor.editSimplePage("", update);
+    wikiManagement.publishPageWhenEditPage();
+    wikiManagement.saveAddPage();
+
+    info("Check email notification and link's format of the page");
+    // goToMail(EMAIL_ADDRESS2, EMAIL_PASS);
+    // wikiManagement.checkEmailNotification(title);
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/WikiBasicActionOtherActionsTestIT.java]: To remove

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/ManageLogInOut.java [hunk 3]
```diff
@@ -111,9 +111,15 @@ public void signInOpenam(String username, String password) {
    */
   public void signInCas(String username, String password) {
     testBase.getExoWebDriver().getWebDriver();
+<<<<<<< ca2b0a624e755255b415d6e6b9216cf8e877171a
```
Comment (thomasdelhomenie) [inline selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/ManageLogInOut.java]: It probably does not compile :smile:

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/social/SpaceSettingManagement.java [hunk 4]
```diff
@@ -80,17 +81,20 @@ public void goToMemberTabInSpaceSettingTab() {
...
+    info("--Search user " );
+    ELEMENT_INPUT_INVITE_USER.click();
+    $(byXpath("//*[@id=\"UIUserInvitation\"]/div[2]/div[2]/div[1]/div[1]/input")).setValue(userName).pressEnter();
+    $(byText(fullName)).waitUntil(Condition.appears,Configuration.timeout);
+    $(byXpath("//*[@id=\"UIUserInvitation\"]/div[2]/div[2]/div[1]/div[1]/input")).pressEnter();
+    $(ELEMENT_SPACE_BTN_INVITE).click();
 
     info("Select a user");
-    evt.click(ELEMENT_SPACE_SELECT_USER_IN_FORM.replace("{$name}", userName), 2);
+    //evt.click(ELEMENT_SPACE_SELECT_USER_IN_FORM.replace("{$name}", userName), 2);
```
Comment (thomasdelhomenie) [inline selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/social/SpaceSettingManagement.java]: To remove

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/testbase/LocatorTestBase.java [hunk 5]
```diff
@@ -27,6 +27,11 @@
 import org.openqa.selenium.By;
 
 import com.codeborne.selenide.SelenideElement;
+import com.codeborne.selenide.SelenideElement;
+import static  com.codeborne.selenide.Selectors.*;
+import org.openqa.selenium.By;
```
Comment (thomasdelhomenie) [inline selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/testbase/LocatorTestBase.java]: This import already exist.
And others added imports seems useless.

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/ManageLogInOut.java (hunk 3 above)
Comment (ilyeskallala) [reply selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/ManageLogInOut.java]: yes, i removed the conflit

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/WikiBasicActionOtherActionsTestIT.java (hunk 1 above)
Comment (ilyeskallala) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/WikiBasicActionOtherActionsTestIT.java]: ok

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/WikiBasicActionOtherActionsTestIT.java (hunk 2 above)
Comment (ilyeskallala) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/WikiBasicActionOtherActionsTestIT.java]: removed

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/social/SpaceSettingManagement.java (hunk 4 above)
Comment (ilyeskallala) [reply selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/social/SpaceSettingManagement.java]: removed

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/testbase/LocatorTestBase.java (hunk 5 above)
Comment (ilyeskallala) [reply selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/testbase/LocatorTestBase.java]: code formatted

**Pull request merged**